docs/docs_source/build_test_ship/shipping.rst

```diff
@@ -0,0 +1,146 @@
+
+..  _go live:
+..  _shipping:
+
+Shipping your Office Online integration
+=======================================
+
+..  spelling::
+
+    rollout
+    Rollout
+
+Once you believe you are ready to ship your integration, you should contact Microsoft and explain that you believe
```

Comment:
How about starting with something like "Wonderful!  We're almost done!" to give it a bit more conversational and happy feel to it? :)